R/calc-limits.R

```diff
@@ -291,6 +294,10 @@ calc_limits <- function(x, by = NULL, term = "long", dates = NULL, keep_limits =
   term <- tolower(term)
   if (!term %in% c("long", "short", "long-daily")) stop("term must be \"long\" or \"short\" or \"long-daily\"")
 
+  if(!use %in% limits$Use) stop("use must match a Use in the limits table")
+  limits <- dplyr::filter_(limits, ~Use %in% use)
```

Comment:
As dplyr has deprecated the underscore versions of the verbs, I'd prefer to use normal `filter` and the `.data$` pronoun. Also, do you want a user to be able to pass more than one `use`? If no, I think `==` would be better than `%in%`

```r
  limits <- dplyr::filter(limits, .data$Use == use)
```

Reply:
As FYI: @atillmanns  and I are adding all water quality guideline uses into the limits.csv, building on the current freshwater life use. There will now be uses for: Freshwater Life, Wildlife, Livestock, Irrigation, Marine Aquatic Life, Drinking Water, Recreation.

Reply:
Sounds good @ateucher. I was using the underscore to be consistent with the rest of the code, but will update. And yes, that should be `Use == use` as we are only accepting one use. @HeatherGranger this is just in reference to what the user can provide to the function. In general this new functionality will address multiple use values in limits table.

Reply:
I've made those changes and pushed @ateucher. Let me know if you need me to do anything else.

Reply:
Looks great, thanks @sebdalgarno!